packages/amplication-client/src/Resource/code-view/CodeViewExplorerTree.tsx

```diff
@@ -113,6 +132,8 @@ const CodeViewExplorerTree = ({
                   file={child}
                   key={child.path + JSON.stringify(child.children)}
                   onSelect={handleNodeClick}
+                  currentFolder={selectedFolder.path}
+                  isLoading={selectedFolder.path === child.path}
```

Comment:
please explain this logic - why "selectedFolder.path === child.path" means that  the node is loading?

Reply:
The logic selectedFolder.path === child.path is used to determine whether a specific folder node is currently in a loading state.

Reply:
In case a user expands a second folder while the content of the first folder is still loading, will that still work? 
I am not sure why this condition necessarily means that it is a loading state